File: LeetCode/Easy/1598-crawler-log-folder/1598-crawler-log-folder.java
class Solution {
    public int minOperations(String[] logs) {
        int ret = 0;
        
        for (String log : logs){
            if (log.charAt(0) == '.') {
                if (log.charAt(1) == '.')
                    ret = Math.max(0, ret - 1);
            }
            else ++ret;
        }
        
        return ret;
    }
}

File: LeetCode/Easy/1598-crawler-log-folder/1598-crawler-log-folder.cpp
class Solution {
public:
    int minOperations(vector<string>& logs) {
        int ret = 0;
        
        for (string log : logs){
            if (log[0] == '.') {
                if (log[1] == '.')
                    ret = max(0, ret - 1);
            }
            else ++ret;
        }
        
        return ret;
    }
};